tools/runners/run-test-suite.sh

```diff
@@ -182,8 +187,7 @@ do
 
         failed=$((failed+1))
     else
-        test $VERBOSE && echo "[$tested/$TOTAL] $cmd_line: $PASS"
-
+        test $VERBOSE && printf "[%4d/%4d] %bPASS: %s%b\n" "$tested" "$TOTAL" "$TERM_GREEN" "$testfile" "$TERM_NORMAL"
```

Comment:
I'm just realizing that you've rewritten `$PASS` to `PASS` here. That's definitely not right. `$PASS` may be `PASS` or `PASS (XFAIL)`, and that has been determined quite some lines above.